### ISSUE TITLE
OrderItem populate_from_cart_item() does not use the cart for the unit_price

### ISSUE THREAD

**Issue by racitup** [awesto/django-shop#635]
I have noticed a minor issue when an order item is created from a cart item.
The unit_price is updated directly by calling `get_price()` again after the cart item unit_price has previously been calculated (3 times by cart.update()) through the cart modifiers. Depending upon the implementation this may or may not be what is desired.

I for example have a get_price() method which by default returns the price in the local customer currency. Yet my cart modifiers maintain the order in the shop local currency. Therefore the cart item unit_price will not be the same as the result from get_price().

The order should be created from exactly what the customer has seen when purchasing, so the order should be populated directly from the cart object.

```
def populate_from_cart_item(self, cart_item, request):
        """
        From a given cart item, populate the current order item.
        If the operation was successful, the given item shall be removed from the cart.
        If a CartItem.DoesNotExist exception is raised, discard the order item.
        """
        if cart_item.quantity == 0:
            raise CartItemModel.DoesNotExist("Cart Item is on the Wish List")
        self.product = cart_item.product
        # for historical integrity, store the product's name and price at the moment of purchase
        self.product_name = cart_item.product.product_name
        self.product_code = cart_item.product_code
        self._unit_price = Decimal(cart_item.product.get_price(request))   ###  <-- HERE
        self._line_total = Decimal(cart_item.line_total)
        self.quantity = cart_item.quantity
        self.extra = dict(cart_item.extra)
        extra_rows = [(modifier, extra_row.data) for modifier, extra_row in cart_item.extra_rows.items()]
        self.extra.update(rows=extra_rows)
```

**Comment by jrief** [comment]
@racitup Thanks for reporting.
This has been fixed in the ``releases/0.11`` branch; is that the branch you are using?
Otherwise I will backport that fix onto the master branch.

btw., this was a legacy issue. In former versions of django-SHOP, the ``cart_item`` did not keep the product's unit price.

**Comment by racitup** [comment]
Cool thanks, I'm still on 0.10.2. Do you know when 0.11 will be ready for release? I'm hoping to deploy soon. Also, will either of my PRs make it in do you think?
